cg/meta/clean/flow_cell_run_directories.py

```diff
@@ -104,15 +103,42 @@ def archive_sample_sheet(self) -> None:
             return
         LOG.info("Sample sheet found!")
         hk_bundle: Bundle = self.hk.bundle(name=self.id)
-        hk_tags: List[str] = [SequencingFileTag.ARCHIVED_SAMPLE_SHEET, self.id]
+        hk_tags: List[str] = [
+            self.id,
+        ]
         if hk_bundle is None:
             LOG.info(f"Creating bundle with name {self.id}")
-            self.hk.create_new_bundle_and_version(name=self.id)
-        elif self.hk.get_file_from_latest_version(bundle_name=hk_bundle.name, tags=hk_tags):
-            LOG.warning("Sample sheet already included!")
-            return
+            hk_bundle = self.hk.create_new_bundle_and_version(name=self.id)
+        sample_sheets_from_latest_version: Optional[
+            List[File]
+        ] = self.sample_sheets_from_latest_version(hk_bundle_name=hk_bundle.name)
+        if sample_sheets_from_latest_version:
+            for file in sample_sheets_from_latest_version:
+                if file.is_included:
+                    LOG.warning("Sample sheet already included!")
+                    return
+        hk_tags.append(SequencingFileTag.ARCHIVED_SAMPLE_SHEET)
         self.hk.add_and_include_file_to_latest_version(
             bundle_name=self.id,
             file=self.sample_sheet_path,
             tags=hk_tags,
         )
+
+    def sample_sheets_from_latest_version(self, hk_bundle_name: str) -> List[File]:
```

Comment:
function name should start with a verb
```suggestion
    def get_sample_sheets_from_latest_version(self, hk_bundle_name: str) -> List[File]:
```